src/steps/networkTunnel/index.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  getRawData,
+  IntegrationStep,
+  IntegrationStepExecutionContext,
+} from '@jupiterone/integration-sdk-core';
+
+import { getOrCreateAPIClient } from '../../client';
+import { IntegrationConfig } from '../../config';
+import { Site } from '../../types';
+import { Entities, Relationships, Steps } from '../constants';
+import {
+  createNetworkTunnelEntity,
+  createSiteTunnelRelationship,
+} from './converter';
+
+export async function fetchNetworkTunnels({
+  instance,
+  jobState,
+  logger,
+}: IntegrationStepExecutionContext<IntegrationConfig>) {
+  const siteLookup: { [key: number]: string } = {};
+  const apiClient = getOrCreateAPIClient(instance.config, logger);
+
+  await jobState.iterateEntities({ _type: Entities.SITE._type }, (site) => {
+    const siteRawData = getRawData(site) as Site;
```

Comment:
I believe `getRawData` takes a generic parameter
```suggestion
    const siteRawData = getRawData<Site>(site);
```